app/src/main/java/com/example/maciej/smim/MultiplayerGameActivity.kt

```diff
@@ -319,4 +323,9 @@ class MultiplayerGameActivity : AppCompatActivity() {
         var whichPlayerTurn: Int = 0
         var index: Int = 0
     }
+
+    override fun onStop() {
```

Comment:
It can't be _onStop_ function here, because it is invoked after changing screen orientation. _onBackPressed_ works correct.